cc/dual_net/lite_dual_net.cc

```diff
@@ -123,33 +176,26 @@ void LiteDualNet::RunMany(std::vector<const BoardFeatures*> features,
   MG_CHECK(batch_size <= input_->dims->data[0]);
 
   // TODO(tommadams): Make BoardFeatures a uint8_t array and memcpy here.
-  auto* data = input_->data.uint8;
+  const auto input_params = input_->params;
   for (size_t j = 0; j < features.size(); ++j) {
     const auto& board = *features[j];
     for (size_t i = 0; i < board.size(); ++i) {
       // TODO(csigg): Apply dequantization parameters?
-      data[j * kNumStoneFeatures + i] = static_cast<uint8_t>(board[i]);
+      feature_data[j * kNumStoneFeatures + i] =
+          Convert<T>(input_params, board[i]);
```

Comment:
Unlike before, this applies the params to the input as well. Could you please confirm that this is what we would want?

Reply:
I think so. I checked one of our tflite models and it had `scale == 1` and `zero_point == 0`, so the conversion is a no-op in our case. Better safe than sorry though :)